.github/file-filters.yml

```diff
@@ -62,6 +64,9 @@ backend: &backend
 acceptance: &acceptance
   - *backend
   - *frontend
+  # This is verbose because backend_build_changes includes it, however,
+  - '.github/actions/setup-sentry/action.yml'
+  - '.github/workflows/acceptance.yml'
```

Comment:
Changes to `acceptance.yml` will cause acceptance jobs to execute.

We already have a change like this for frontend changes.